src/components/Footer/FooterExtendedNavList/FooterExtendedNavList.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react'
+import classnames from 'classnames'
+
+export type ExtendedNavLinksType = React.ReactNode[][]
+
+type FooterExtendedNavListProps = {
+  /* 
+    Multidimensional array of grouped nav links. Sub-arrays are column sections, first element is used as a heading.
+  */
+  nestedLinks: ExtendedNavLinksType
```

Comment:
Would it be a bit more concise to just have the `nestedLinks: React.ReactNode[][]` ?

Reply:
Hey maybe! I used a named type to export it and use it in multiple files. But that might be overkill since this isn't a complex type.